packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -97,6 +97,7 @@ export function generateRuntimeOptions(
 		enableOpReentryCheck: [true],
 		// Compressed payloads exceeding this size will be chunked into messages of exactly this size
 		chunkSizeInBytes: [204800],
+		enableRuntimeIdCompressor: [undefined],
```

Comment:
You should enable id compressor in the stress tests at some point in time. Maybe create an issue to do this later?
I don't see any harm in enabling it now since the test won't use it at all. It will give you converage of the fact that it doesn't break anything if unused.

Reply:
Enabled in latest commit